build/Win2D.cpp.props

```diff
@@ -45,7 +53,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```

Comment:
WindowsTargetPlatformVersion is usually a project-wide setting, and it often has follow-on effects. Typically it is changed by itself all at once and the change followed by extensive testing. I recommend this because it allows SDK version differences to be dealt with separate from architecture differences.

In this case, the README states that it is 17134 and Win2D itself is built for 17134:
    build\nuget\Win2D.common.targets:5:    <Win2DMinPlatformVersion>10.0.17134.0</Win2DMinPlatformVersion>

The way to build without changing the SDK version is to install the older SDK using the "Individual Components" pane of the Visual Studio Installer.